### ISSUE TITLE
mixins-before-declarations should not raise warning with top-level mixins

### ISSUE THREAD

**Issue by BPScott** [sasstools/sass-lint#227]
Currently `@include`ing a mixin at the top level (i.e. not inside a selector) incorrectly raises a 'mixins-before-delcarations' warning, when this should be acceptable (there are no properties before it!).

To reproduce, add the following to the bottom of [mixins-before-declarations.scss](https://github.com/sasstools/sass-lint/blob/develop/tests/sass/mixins-before-declarations.scss):

```scss
@include hello;

@media (min-width: 50em) {
  @include hello;
}

@supports (display: flex) {
  @include hello;
}

@at-root {
  @include hello;
}
```

I would expect that this would raise no additional linting errors.

**Comment by shaneog** [comment]
:+1:
